README.md

```diff
@@ -384,3 +385,47 @@ data:
 ```
 
 When starting kube-httpcache, remember to set the `--backend-watch=false` flag to disable watching the (non-existent) backend endpoints.
+
+## Helm Chart installation
+
+Using [HELM](chart/) to rollout an instance of kube-httpcache.
+
+Ensure your defined frontend and backend services have a port
+name `http`:
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: backend-service
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  type: ClusterIP
+```
+
+An ingress points to the kube-httpcache service which cached 
+your backend service:
+
+```
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: example-ingress
+spec:
+  rules:
+  - host: www.example.com
+    http:
+      paths:
+      - backend:
+          serviceName: kube-httpcache
+          servicePort: 80
+        path: /
+```
+
+Look at the [Varnish Go Template file](chart/varnish.tpl) to define
+your own Varnish cluster rules or load the file as initContainer.
```

Comment:
I feel that given the complexity of the Helm chart (and the amount of required customization), a bit more verbosity might be warranted for the documentation, in this case.

Reply:
more explained the 2 use cases with configmap and file.